parm/config/gefs/config.wave

```diff
@@ -0,0 +1,202 @@
+#! /usr/bin/env bash
```

Comment:
Can we just link to `config.wave` from `parm/config/gfs`?  Is there any distinction for wave between gfs and gefs for waves?

Reply:
Not sure if we will need to use different settings for GEFS. Can certain do that (and for aero/ice/ocn) until such time as we find we need to have different settings. Will leave it up to you.

Reply:
Done.